src/elf.rs

```diff
@@ -814,14 +760,19 @@ impl<'a> Elf<'a> {
 
         // alignment required for below
         let mut symtab_offset = strtab_offset + self.sizeof_strtab as u64;
-        let sizeof_symbol = Symbol::size(self.ctx.container) as u64;
-        Self::align(&mut symtab_offset, sizeof_symbol);
-        let mut reloc_offset = symtab_offset + sizeof_symtab as u64;
-        let sizeof_relocation = Relocation::size(true, self.ctx) as u64;
-        Self::align(&mut reloc_offset, sizeof_relocation);
+        let symtab_align = self.ctx.size() as u64;
```

Comment:
Ditto here, this is less readable than explicit `Symbol::size`; I want to know which sizeof I'm getting